### ISSUE TITLE
Style guide

### ISSUE THREAD

**Issue by SimonLab** [FAC-GM/refonte#2]
We need some style! http://hapijs.com/styleguide and use jslint to validate the code at each commit!

**Comment by SimonLab** [comment]
http://www.sitepoint.com/comparison-javascript-linting-tools/ comparison jslint vs jshint vs eslint

**Comment by iteles** [comment]
👍🏻

**Comment by nelsonic** [comment]
ESLint :+1:

**Comment by SimonLab** [comment]
http://eslint.org/docs/user-guide/getting-started

**Comment by SimonLab** [comment]
Also as part of the new style guide, we need to have a set of features from es6 (watch [Crockford video](https://www.youtube.com/watch?v=PSGEjv3Tqo0)) **to use on the server side only**

**Comment by SimonLab** [comment]
What about https://github.com/feross/standard instead of eslint?

**Comment by iteles** [comment]
Why does everyone keep trying to take away my semicolons?

**Comment by SimonLab** [comment]
Yes I know I was a bit worry about this one I need to read/watch the links to have a better idea
- https://www.youtube.com/watch?v=gsfbh17Ax9I
- http://inimino.org/~inimino/blog/javascript_semicolons
- http://blog.izs.me/post/2353458699/an-open-letter-to-javascript-leaders-regarding

**Comment by nelsonic** [comment]
@SimonLab We've been using [`semistandard`](https://github.com/Flet/semistandard) for our projects, but I'm about to move away from it because the latest update which introduces rules I don't think are logical.

**Comment by SimonLab** [comment]
@nelsonic Do you have an idea of which one you're going to choose next?

**Comment by nelsonic** [comment]
Yep. https://github.com/nelsonic/nelsonic.github.io/issues/193 / https://github.com/dwyl/goodparts
I just don't have time in the next couple of days to do it... 
what research have you done?
